boilerplate/openshift/golang-osd-operator/ensure.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -euo pipefail
+set -eo pipefail
```

Comment:
Note to reviewers: `-u` was kicking out `$BOILERPLATE_SET_X` when it was unset (the default). If you just hate this, I can default it to the empty string instead.